tests/api_tests/base/test_views.py

```diff
@@ -47,6 +53,9 @@ def test_view_classes_have_minimal_set_of_permissions_classes(self):
         view_modules = [name for _, name, _ in pkgutil.iter_modules(['api'])]
         for module in view_modules:
             for name, view in inspect.getmembers(sys.modules['api.{}.views'.format(module)], inspect.isclass):
+                if getattr(view, 'permissions_exempt', False):
```

Comment:
Okay, so because it's a base_view, it doesn't need to have the base permissions, as it will never be a view of its own, yes? There'll always be a subclass (or a mixed class) that has this and other things, and that child/mix will have the permissions checked. Something like that?